factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/registry/AbstractSchemaRegistry.java

```diff
@@ -93,7 +93,7 @@ public void fetchInitial() {
 
             Stopwatch sw = Stopwatch.createStarted();
             log.info("Registry update started");
-            refresh();
+            indexFetcher.fetchIndex().ifPresent(this::process);
```

Comment:
You dont want to use `refresh()`because of the mutex?

Reply:
refresh is an extension point for impls to overwrite. for instance classpath: does not need a refresh.

Reply:
Okay, so timings have to be added in both places

Reply:
wondering, if initial timing (which is in the logs and an obstacle to startup) is at all helpful.

Reply:
Also true 🤔